lib/ClimaLandSimulations/src/fluxnet_simulation.jl

```diff
@@ -8,71 +37,44 @@ Run ClimaLand at a fluxnet site.
 function fluxnet_simulation(
     site_ID;
     FT = Float64,
-    path_to_sim = joinpath(
-        ClimaLandSimulations_dir,
-        "src",
-        "fluxnet",
-        "$site_ID",
-        "$(site_ID)_simulation.jl",
+    site_setup_out = site_setup(site_ID),

```

Comment:
we could say "setup = site_setup()", "domain = site_domain(),"... etc instead of `site_setup_out`?

Reply:
what is the difference between `setup` and `config`? :)

Reply:
(could use more descriptive names?)

Reply:
I renamed as suggested, not sure about `setup` and `config` - I think it could possibly be done from the same file, we could definitely use some reorganization (it's on the to do list)